app/src/androidTest/java/com/soen490chrysalis/papilio/LoginActivityUITest.kt

```diff
@@ -169,79 +149,4 @@ class LoginActivityUITest
         // There should be no errors displayed
         Espresso.onView(ViewMatchers.withId(R.id.user_password)).check(ViewAssertions.matches(hasNoErrorText()))
     }
-
-    @Test
-    fun verifyRedirectionOnLoginWithEmail()
```

Comment:
These tests should not have been deleted since they are part of the authentication part!